src/spectral.ts

```diff
@@ -11,11 +13,28 @@ import {
   isOpenApiv3,
 } from '@stoplight/spectral';
 
+import * as SpectralPackage from '../node_modules/@stoplight/spectral/package.json';
```

Comment:
This cannot be statically imported, because this might not exist. You need to replace this with an `IOEither` with a require call

Reply:
@XVincentX I've _tried_ to apply your hints (Not that they were unclear, but rather because I'm not really used to `fp-ts`. Hence `tried`).

Could you please check that I've correctly understood the requirements?